erc20-bridge-token/contracts/ERC20Locker.sol

```diff
@@ -0,0 +1,61 @@
+pragma solidity ^0.8;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "rainbow-bridge-sol/nearprover/contracts/INearProver.sol";
+import "rainbow-bridge-sol/nearprover/contracts/ProofDecoder.sol";
+import "rainbow-bridge-sol/nearbridge/contracts/NearDecoder.sol";
+import "rainbow-bridge-sol/nearbridge/contracts/Borsh.sol";
+
+import "./Locker.sol";
+
+contract ERC20Locker is Locker {
```

Comment:
What's the purpose of this contract? Looks like a duplication of the old contract used for ETH->NEAR connector.

Reply:
Probably it was used for early prototyping but now it's not used and deleted.